### ISSUE TITLE
Selecting Load Balancing method on VS CRD

### ISSUE THREAD

**Issue by skenderidis** [F5Networks/k8s-bigip-ctlr#2224]
#### Title
The user should be able to define the LB method for the VirtualServer CRD similar to what we can do with Ingress

#### Description
The user should be able to define the LB method for the VirtualServer CRD similar to what we can do with Ingress

#### Actual Problem
There are many cases that Round Robin is not the most effective algorithm for LB. 

#### Solution Proposed
On the Pool spec we need to give the option to define the LB method. This needs to take place in the pool as you might want to have different LB methods per pool (as it is done on the BIGIP)

![image](https://user-images.githubusercontent.com/11005568/152629138-83e22752-4901-4080-84f5-91e5cb8d2a80.png)

**Comment by trinaths** [comment]
Created CONTCNTR-3146 for internal tracking.

**Comment by trinaths** [comment]
Duplicate of https://github.com/F5Networks/k8s-bigip-ctlr/issues/2202

Closing this issue.

**Comment by jmorello-mdsol** [comment]
Has this feature been added yet or is it scheduled for a release soon? 
I don't see it in the customResourceDefinitions v2.8.0 for VS and TS, but maybe I missed it. 
This is something we require and are waiting on the feature before we start using CIS in CRD mode.
